### PR TITLE
Improving the UI a bit

### PR DESCRIPTION
**Pull request by lfilho**

### CODE REVIEW

File: frontend/components/charts/team-size.js [hunk 1]
```diff
@@ -77,8 +77,9 @@ function createChart (series) {
   let chart = new Highcharts.Chart({
     chart: {
       renderTo: 'team-size-chart',
+      backgroundColor: 'transparent',
       type: 'area',
-      zoomType: 'x'
+      zoomType: 'xy'
```
Comment (lfilho) [inline frontend/components/charts/team-size.js]: I figured we could use the y zoom as well when the areas are to thin...

**Comment by lfilho** [comment]
image

**Comment by lfilho** [comment]
image

**Comment by lfilho** [comment]
image

**Comment by lfilho** [comment]
image

File: static/img/x.svg [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (lfilho) [inline static/img/x.svg]: I reversed engineered the `x.png` into a `x.svg` but one of the corners got a bit rounded up. If you have a proper svg, feel free to update it :)

**Comment by lfilho** [comment]
image

**Comment by joshwnj** [comment]
lookin good! I think we'll still change a few things after merging, but this is a great step in the right direction.

**Pull request merged**